crates/sui-core/src/authority/authority_test_utils.rs

```diff
@@ -527,7 +527,11 @@ pub async fn publish_package_on_single_authority(
             .additional_named_addresses
             .insert(addr_name.to_string(), AccountAddress::from(obj_id));
     }
-    let modules = build_config.build(path).unwrap().get_package_bytes(false);
```

Comment:
The `get_package_bytes` here takes the `with_unpublished_dependencies` arg, so moved that into a variable to be able to pass it both to `build` and to `get_package_bytes`